src/core/labeling/qgslabelfeature.h

```diff
@@ -274,6 +274,18 @@ class CORE_EXPORT QgsLabelFeature
      */
     void setDistLabel( double dist ) { mDistLabel = dist; }
 
+    /**
+     * Applies to on-line "curved" placement strategy for linestring features.
+     * Distance of the label from the feature (in map units)
+     */
+    double originalDistLabel() const { return mOriginalDistLabel; }
```

Comment:
How would you feel about flipping this, and adding "adjustedLabelDistance()" instead? Then the above/below line code would be changed to use the adjusted distance instead of distLabel(). I think that approach would be much more readable...